app/components/app_patient_card_component.rb

```diff
@@ -46,5 +46,26 @@ def initialize(
 
   def show_school_and_year_group = patient.show_year_group?(team: current_team)
 
-  def important_notices = ImportantNotices.call(patient:)
+  def important_notices
+    latest_important_notices(patient:)
+  end
+
+  def latest_important_notices(patient:)
```

Comment:
Wondering if we can just combine these two in to a single method, since we're only calling `latest_important_notices` in one place.